src/modules/dashboard/sagas/actions.ts

```diff
@@ -241,10 +242,108 @@ function* createMoveFundsAction({
   }
 }
 
+function* createMintTokensAction({
+  payload: { colonyAddress, colonyName, nativeTokenAddress, amount },
+  meta: { id: metaId, history },
+  meta,
+}: Action<ActionTypes.COLONY_ACTION_MINT_TOKENS>) {
+  let txChannel;
+  try {
+    const apolloClient = TEMP_getContext(ContextModule.ApolloClient);
+
+    if (!amount) {
+      throw new Error('Amount to mint not set for mintTokens transaction');
+    }
+
+    txChannel = yield call(getTxChannel, metaId);
+
+    // setup batch ids and channels
+    const batchKey = 'mintTokens';
+    const mintTokens = {
+      id: `${metaId}-mintTokens`,
+      channel: yield call(getTxChannel, `${metaId}-mintTokens`),
+    };
+    const claimColonyFunds = {
+      id: `${metaId}-claimColonyFunds`,
+      channel: yield call(getTxChannel, `${metaId}-claimColonyFunds`),
+    };
+
+    // create transactions
+    yield fork(createTransaction, mintTokens.id, {
+      context: ClientType.ColonyClient,
+      methodName: 'mintTokens',
+      identifier: colonyAddress,
+      params: [amount],
+      group: {
+        key: batchKey,
+        id: metaId,
+        index: 0,
+      },
+      ready: false,
+    });
+    yield fork(createTransaction, claimColonyFunds.id, {
+      context: ClientType.ColonyClient,
+      methodName: 'claimColonyFunds',
+      identifier: colonyAddress,
+      params: [nativeTokenAddress],
+      group: {
+        key: batchKey,
+        id: metaId,
+        index: 1,
+      },
+      ready: false,
+    });
+
+    yield takeFrom(mintTokens.channel, ActionTypes.TRANSACTION_CREATED);
+    yield takeFrom(claimColonyFunds.channel, ActionTypes.TRANSACTION_CREATED);
+    yield put({ type: ActionTypes.COLONY_ACTION_MINT_TOKENS_SUBMITTED });
```

Comment:
This is not wired anywhere. What does it do ?

Reply:
Right. I found out what it used to do. 

In the previous iteration, it was used to close the modal before the gas station opened. We're not doing that now, so this is pointless.

Please remove this as well as the redux action.

But in the future please investigate better the code you are copy-pasting or refactoring.